blog-post.html

```diff
@@ -1,12 +1,80 @@
+  <h5> Lorem ipsum dolor sit amet consectetur, adipisicing elit. Iste obcaecati, facere est ad iure culpa earum, sapiente beatae ipsum ducimus, et doloribus. Odio corrupti eius autem ipsam aliquid doloremque beatae labore, necessitatibus velit? Optio totam architecto harum, laborum corrupti fugit nobis accusantium, aspernatur alias eveniet earum praesentium hic. Recusandae iusto a eum voluptatibus, corporis laudantium hic. Illo impedit laudantium esse. Iure recusandae ipsa ad corrupti aliquid! Impedit eos eaque earum eum placeat qui neque dolor possimus quibusdam illum dolorem, tempora quasi minus distinctio! Id dolor at perspiciatis molestias.Iure recusandae ipsa ad corrupti aliquid! Impedit eos eaque earum eum placeat qui neque dolor possimus quibusdam illum dolorem, tempora quasi minus distinctio! Id dolor at perspiciatis molestias.  </h5>
+
+  <h3> Why Hello There</h3>
+      
+  <h5>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus aperiam quam autem, natus consequuntur facere doloribus quos, illum in, eaque reprehenderit. Alias possimus quos, nam labore praesentium repellat quibusdam ducimus veritatis esse dolore tempore rerum, hic aperiam voluptatem quas quis consequuntur architecto molestias, voluptate tempora dolor debitis. Iure recusandae ipsa ad corrupti aliquid! Impedit eos eaque earum eum placeat qui neque dolor possimus quibusdam illum dolorem, tempora quasi minus distinctio! Id dolor at perspiciatis molestias. </h5>
+  <h5> Lorem ipsum dolor sit amet consectetur, adipisicing elit. Iste obcaecati, facere est ad iure culpa earum, sapiente beatae ipsum ducimus, et doloribus. Odio corrupti eius autem ipsam aliquid doloremque beatae labore, necessitatibus velit? Optio totam architecto harum, laborum corrupti fugit nobis accusantium, aspernatur alias eveniet earum praesentium hic. Recusandae iusto a eum voluptatibus, corporis laudantium hic. Illo impedit laudantium esse. Iure recusandae ipsa ad corrupti aliquid! Impedit eos eaque earum eum placeat qui neque dolor possimus quibusdam illum dolorem, tempora quasi minus distinctio! Id dolor at perspiciatis molestias.Iure recusandae ipsa ad corrupti aliquid! Impedit eos eaque earum eum placeat qui neque dolor possimus quibusdam illum dolorem, tempora quasi minus distinctio! Id dolor at perspiciatis molestias.Iure recusandae ipsa ad corrupti aliquid! Impedit eos eaque earum eum placeat qui neque dolor possimus quibusdam illum dolorem, tempora quasi minus distinctio! Id dolor at perspiciatis molestias.  </h5>
+    </section>
+    <div class = "mainL">
+      <br><br>
+      <a href="learn-more.html"><button>JUNE 06</button></a>
+         
+          <h4 class = "first" >THIS IS THE TOP STORY</h4>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Repudiandae est eligendi modi soluta eos voluptates nemo dolorem, reprehenderit iure molestias repellat ex libero? Molestiae, facilis?</p>
+       <br><br>

```

Comment:
you need a closing tag for the `<br>` such as `<br></b>` or a self closing tag such as `<br />`, you have a few of these throughout all your html pages